Assets/MixedRealityToolkit/Extensions/BoundsExtensions.cs

```diff
@@ -457,6 +457,88 @@ public static void GetMeshFilterBoundsPoints(GameObject target, List<Vector3> bo
             }
         }
 
+        /// <summary>
+        /// Finds the object-aligned size of a <see cref="UnityEngine.Transform"/> 
```

Comment:
It's important to mention how the bounds are searched: first it tries to find collider on self. If it finds none, then it returns the bounds of the child mesh renderers.

Note: this is not what I would expect. I would expect the behavior to try to find collider on self, and then look for children with collidres. In fact, have a look at BoundingBox.BoundsCalculationMethod and how that's used -- I think this utility method could just use that bounds calculation method to compute the bounds.

Note: might be out of scope for this pr. In that case, I'd recomment just pulling this into ScrollingObjectCollection.

Reply:
I'll move it for now, I agree that it should traverse the hirerarchy of the Transform for all colliders.